build/root/WORKSPACE

```diff
@@ -4,8 +4,9 @@ load("//build:workspace.bzl", "CRI_TOOLS_VERSION")
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "7519e9e1c716ae3c05bd2d984a42c3b02e690c5df728dc0a84b23f90c355c5a1",
-    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/0.15.4/rules_go-0.15.4.tar.gz"),
+    sha256 = "548f2ca9318c6a4393cdb4d80be09c0c85da0ed90eaf72820ac03786b09f7e9b",
+    strip_prefix = "rules_go-36605c479ee09646734278c24c6b4abea5aef9c9",
+    urls = mirror("https://github.com/bazelbuild/rules_go/archive/36605c479ee09646734278c24c6b4abea5aef9c9.tar.gz"),
```

Comment:
I'd prefer to wait for a release (I imagine there will be one in the next day or two)

Reply:
@ixdy Yeah, I agree. If you look at the commits, I've got a placeholder commit for the next version bump, and then a WIP commit to use this HEAD so that we can verify that bazel-test and bazel-build pass.

Reply:
I'll include the new rules_go bump in https://github.com/kubernetes/kubernetes/pull/70456, but I'll reserve the go version bump for this PR.

Reply:
Oh! Okay. Well then I'll wait for bazel-test to pass, then I'll drop those commits from this PR completely.

Reply:
Okay, got a green run on bazel-test (https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70665/pull-kubernetes-bazel-test/66242/).